shared/modules/runtime_module.go

```diff
@@ -0,0 +1,7 @@
+package modules
+
+type Runtime interface {
+	GetConfig() Config
+	GetGenesis() GenesisState
+	ShouldUseRandomPK() bool
```

Comment:
I put it there exactly because there's a semantic conflict between that flag and the privateKey in config.

If this was only used in tests, yeah, we can mock that but currently the client needs a random private key to work for debugging purposes. I concur, something to think about.